### DISCUSSION TITLE
Picking a material form the list

### DISCUSSION THREAD

**Discussion by adrianposteuca** [google/model-viewer#3274] [Q&A] [answered]
Hello all,
I would like to pick a material from the materials list
const [material] = modelViewer.model.materials

I do not know what it returns? A list of int/index or a list of strings?
How can I access material 3 
material[3].pbrMetallicRoughness.setBaseColorFactor(color)
material["inside"].pbrMetallicRoughness.setBaseColorFactor(color)

Thank you

**Comment by samaneh-kazemi** [comment, marked as answer]
Hi @adrianposteuca, model.materials returns a list of objects of type Material. 

`(property) Model.materials: Material[]`

So you can do sth like :
```
let material = modelViewerParameters.model.materials[0];
material.pbrMetallicRoughness.setBaseColorFactor([0.7294, 0.5333, 0.0392]);
```

Checkout our [Change Material Metalness and Roughness Factors](https://modelviewer.dev/examples/scenegraph/#changeMaterial) example
